src/plugins/kibana_usage_collection/server/collectors/saved_objects_counts/kibana_usage_collector.ts

```diff
@@ -80,8 +80,9 @@ export function registerKibanaUsageCollector(
         },
       },
       async fetch({ soClient }) {
+        const indices = await getIndexForTypes(['dashboard', 'visualization', 'search']);
         return {
-          index: kibanaIndex,
+          index: indices[0],
```

Comment:
I couldn't change the shape of the collector's data, so `index` has to stay a single string. In practice it's fine, given all the types counted here are in a single index.

Reply:
IIUC, the `getKibanaSavedObjectCounts()` will aggregate data from all SO indices under the hood.
Then, from the stats perspective, it is fine if we still report them as being part of the `.kibana` index. Correct?
Ping @afharo for awareness